week1/readme.md

```diff
@@ -1,9 +1,87 @@
 ## Introduction to C 
 
 ### Topics
-1. Fucntion 
-2. if/else statements
-3. Loops (for,wile,dowhile)
-4. Operators
-5. Dry Run
-6. Truth table
\ No newline at end of file
+
+
+1. Vs code: 
+The compiler, can be divided into a number of regions; there is a file explorer on the left side to find files. Further, there is a region in the middle called a text editor where you can edit your program. Finally, there is a command line interface, known as a CLI, command line, or terminal window where we can send commands to the computer in the cloud.
+
+2. Code writing: 
+Code can be evaluated upon three axes. First, correctness refers to does the code run as intended Second, design refers to how well is the code designed? Finally, style refers to how aesthetically pleasing and consistent is the code?
+
+3. Functions:
+The statement at the start of the code #include <stdio.h> is a very special command that tells the compile that you want to use the capabilities of library called stdio.h. This allows you, among many other things, to utilize the printf function
+Within the printf function it isvery important to note that without certain smybols like the the round brackets after the printf function containg the double quotes at the start and end of the and within thore double quotes we write what we want to print eg; Hello, world followed by a \n to get the next line and ending with the semicolon
+Execute make hello and then Typing ./hello in the terminal window to run the program.
+
+4. Variables
+#include <cs50.h> should be added to the top of your code, this deals with taking thw input from the user. For taking a string from the user, get_string function is used. Then, the variable answer is passed to the printf function. %s tells the printf function to prepare itself to receive a string follewed by the name of that variable.
+i.e answer; is a special holding place we call a variable. answer is of type string and can hold any string within it. There are many data types, such as int, bool, char, long, float and many others.
+
+4. Dry run:
+A dry run (or practice run) is a software testing process used to make sure that a system works correctly and will not result in severe failure.At the most basic level, a dry run test involves a programmer reading the code to make sure it works without actually running.I learned the dry run by executing my loop code within the dry run  to make sure that the loop is working fine.
+
+5. Truth table:
+Mathematics normally uses a two-valued logic: every statement is either true or false, by using operations like: Conjunction (AND), disjunction (OR), negation (NOT), 
+
+6. Conditionals
+    if/ else/ else if
+    if x is greater than y, if true? the condition if will run or if false i will do something else if that condition is not met.
+    ```javascript
```

Comment:
it is not javascript it is c so it should be something like this 
```c
```

Reply:
I will make sure the next time